### DISCUSSION TITLE
手机认证代理微件问题

### DISCUSSION THREAD

**Discussion by jbycc** [natfrp/wiki#437] [使用问题]
![Screenshot_20240209_214004](https://github.com/natfrp/wiki/assets/120253325/ad0487f5-7bd7-41c4-bff6-0792326fd51f)
如图，无法使用微件。点击无反应。

隧道已启动，这个设备未认证，正常的认证方式可以使用。

配置改过，有设置认证后启动应用。

**Comment by fakeboboliu** [comment]
请问您具体的系统版本是？

看起来是小米？新版 MIUI 和 澎湃OS 出于推广付费小组件的需求，对安卓小组件可能新增了一些限制？

我们会尽快测试并进行维修。

**Reply by jbycc** [reply]
鸿蒙系统，手机是nova9

**Reply by fakeboboliu** [reply]
> 鸿蒙系统，手机是nova9

我借到了一台 P60，鸿蒙 4.0.0，并没有出现任何问题。

能否请您尝试重装应用尝试一下怎样复现这个问题？

**Reply by jbycc** [reply]
抱歉，最近没有关注GitHub的消息。
重装应用后在手机桌面打开鸿蒙的桌面小组件，找到auth，扫码配置，原封不动添加组件后就会出现问题。
![Screenshot_20240225_172941](https://github.com/natfrp/wiki/assets/120253325/46050048-0834-4552-9a62-79ae487bc573)
扫码添加的配置

**Reply by fakeboboliu** [reply]
> 原封不动添加组件后就会出现问题

很抱歉，我这样操作很多次也不会出现问题，我没有鸿蒙相关经验所以目前很难分析这个问题来源。
